sdk/storage/Azure.Storage.Blobs/api/Azure.Storage.Blobs.net6.0.cs

```diff
@@ -56,6 +56,7 @@ public partial class BlobClientOptions : Azure.Core.ClientOptions
         public Azure.Storage.Blobs.Models.CustomerProvidedKey? CustomerProvidedKey { get { throw null; } set { } }
         public bool EnableTenantDiscovery { get { throw null; } set { } }
         public string EncryptionScope { get { throw null; } set { } }
+        public Azure.Storage.ExpectContinueOptions ExpectContinueBehavior { get { throw null; } set { } }
```

Comment:
Do regular developers know what `ExpectContinue` is?  This feels more like the "how" not the "why."  Should we come up with a more intuitive name like `(Cautious|Preflight|...)(Upload|Transfer|...)`?  Ideally a better name would also let us follow our usual pattern of `public FooOptions Foo { get; set; }` for *Options types in *ClientOptions.

Does this need to be top-level or should it be nested in `StorageTransferOptions`?

Reply:
I'm not really sure what a good name would be, given it's a very specific header with understood behavior. Even if we come up with a better top level name, I would still expect we use this language for the individual properties of the class.

I don't think this can go in `StorageTransferOptions`. That is only available on a subset of our transfers, whereas we want this on any request with a content body.